releases/release-6.5.10.md

```diff
@@ -0,0 +1,150 @@
+---
+title: TiDB 6.5.10 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 6.5.10.
+---
+
+# TiDB 6.5.10 Release Notes
+
+Release date: June xx, 2024
+
+TiDB version: 6.5.10
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.5/production-deployment-using-tiup)
+
+## Compatibility changes
+
+<!-- tw:@hfxsd 1-->
+- (dup): release-7.5.2.md > Compatibility changes - In earlier versions, when processing a transaction containing `UPDATE` changes, if the primary key or non-null unique index value is modified in an `UPDATE` event, TiCDC splits this event into `DELETE` and `INSERT` events. Starting from v6.5.10, when using the MySQL sink, TiCDC splits an `UPDATE` event into `DELETE` and `INSERT` events if the transaction `commitTS` for the `UPDATE` change is less than TiCDC `thresholdTS` (which is the current timestamp fetched from PD when TiCDC starts replicating the corresponding table to the downstream). This behavior change addresses the issue of downstream data inconsistencies caused by the potentially incorrect order of `UPDATE` events received by TiCDC, which can lead to an incorrect order of split `DELETE` and `INSERT` events. For more information, see [documentation](https://docs.pingcap.com/tidb/v6.5/ticdc-behavior-change#mysql-sink). [#10918](https://github.com/pingcap/tiflow/issues/10918) @[lidezhu](https://github.com/lidezhu)
+- Must set the line terminator when using TiDB Lightning `strict-format` to import CSV files [#37338](https://github.com/pingcap/tidb/issues/37338) @[lance6716](https://github.com/lance6716)
```

Comment:
The same issue also exists in https://docs.pingcap.com/tidb/stable/release-7.5.2#compatibility-changes. The descriptions are slightly different. Wondering whether we should keep them consistent.

Reply:
Exclude SPLIT_FILE  because it belongs to IMPORT INTO that is not available in v6.5 yet.